src/contribution-guide/suggesting-an-improvement.md

```diff
@@ -0,0 +1,33 @@
+# Suggesting an improvement
+
+If you notice an issue with Kubewarden documentation or want to suggest new content, then open an issue. You only require access to a [GitHub account](https://github.com/join) and a browser.
+
+In most cases, any new documentation work for Kubewarden begins with an issue in GitHub. The [documentation team for Kubewarden](https://github.com/orgs/kubewarden/teams/kubewarden-documentation) reviews, categorizes and tags them as appropriate. Everybody is welcome to work on the issue, including the reporter, but we request that you assign it to yourself before commencing any work to avoid duplicate efforts.
+
+## Opening an issue
+
+If you want to suggest improvements to existing documentation content or notice an error, then open an issue.
+
+- Click the GitHub Octocat icon on the top. This redirects you to the [documentation repository for Kubewarden](https://github.com/kubewarden/docs).
+- Navigate to the Issues tab and click **New issue**.
+- Describe the issue or suggestion for improvement. The more details you provide, the better!
+- Click Submit new issue.
+- Post submitting, you can either assign it to yourself or wait for a community member to pick it up. Members of the documentation team and from the community might request clarifications before they can take action on your issue, so we'd request you to actively check your issue or [turn on GitHub notifications](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications).
+
+## New content suggestions
+
+If you want to suggest new content, please file an issue following the steps above. Either
+
+- You can choose an existing page in the section you think the content belongs in and click Create an issue.
```

Comment:
```suggestion
- Choose an existing page in the section you think the content belongs in and click **Create an issue**.
```

Reply:
Suggesting to make the instructions parallel (so both options start with action verb with an understood "You").